package/yast2-bootloader.spec

```diff
@@ -46,10 +46,6 @@ Requires:       rubygem(%rb_default_ruby_abi:cfa_grub2) >= 0.5.1
 # lenses are needed here
 Requires:       augeas-lenses
 
-%ifarch %ix86 x86_64
-Requires:       syslinux
-%endif
-
```

Comment:
yes, but it is consistent with other yast modules optional dependencies install only when needed.